pwintools.py

```diff
@@ -18,11 +18,6 @@
 from windows.generated_def.winstructs import *
 import windows.native_exec.simple_x64 as x64
 
-if sys.version_info[0] == 3:
-    xrange = range
-    print("Python 3 is not supported")

```

Comment:
just remove the print, we don't want to break py2 compat

Reply:
I also replaced xrange by range in the code. the the compatibility should not be broken not broken.

Anyway. The compatibility with python2 is absolutely not important. Python2 is obsolete since years. Actually even since 15 years everybody is supposed to write in python3.


You can still debate.